packages/myst-cli/src/process/notebook.ts

```diff
@@ -25,43 +24,36 @@ export async function processNotebook(
   content: string,
 ): Promise<Root> {
   const { log } = session;
-  const { notebook, children } = parseNotebook(JSON.parse(content));
+  const { metadata, cells } = JSON.parse(content) as INotebookContent;
   // notebook will be empty, use generateNotebookChildren, generateNotebookOrder here if we want to populate those
 
-  const language = notebook.language ?? notebook.metadata?.kernelspec.language ?? 'python';
+  const language = metadata?.kernelspec?.language ?? 'python';
   log.debug(`Processing Notebook: "${file}"`);
 
-  const fileFactory = createWebFileObjectFactory(log, session.publicPath(), '_static', {
-    useHash: true,
-  });
+  const cache = castSession(session);
 
   const outputMap: Record<string, MinifiedOutput[]> = {};
 
-  let end = children.length;
-  if (
-    children &&
-    children.length > 1 &&
-    children?.[children.length - 1].content.content.length === 0
-  )
+  let end = cells.length;
+  if (cells && cells.length > 1 && cells?.[cells.length - 1].source.length === 0) {
     end = -1;
+  }
 
-  const items = await children?.slice(0, end).reduce(async (P, item: TranslatedBlockPair) => {
+  const items = await cells?.slice(0, end).reduce(async (P, cell: ICell) => {
     const acc = await P;
-    if (item.content.kind === KINDS.Content) {
-      if (item.content.format === ContentFormatTypes.md)
-        return acc.concat(asString(item.content.content));
-      if (item.content.format === ContentFormatTypes.txt)
-        return acc.concat(`\`\`\`\n${asString(item.content.content)}\n\`\`\``);
+    if (cell.cell_type === CELL_TYPES.markdown) {
```

Comment:
Are these cell type enums defined in nbformat?

Reply:
Not super nicely, just as a union of strings type: https://github.com/jupyterlab/jupyterlab/blob/master/packages/nbformat/src/index.ts#L144

Reply:
Sounds good!